examples/transaction.rs

```diff
@@ -23,7 +22,7 @@ async fn get(client: &Client, key: Key) -> Option<Value> {
 
 // Ignore a spurious warning from rustc (https://github.com/rust-lang/rust/issues/60566).
 #[allow(unused_mut)]
```

Comment:
I think this issue should've been solved :)

Reply:
Removed at b7de583